package/up2date.nuspec

```diff
@@ -1,65 +1,30 @@
-    <!-- If the software is cross-platform, attempt to use the same id as the debian/rpm package(s) if possible. -->
     <id>up2date</id>
-    <!-- version should MATCH as closely as possible with the underlying software -->
-    <!-- Is the version a prerelease of a version? https://docs.nuget.org/create/versioning#creating-prerelease-packages -->
-    <!-- Note that unstable versions like 0.0.1 can be considered a released version, but it's possible that one can release a 0.0.1-beta before you release a 0.0.1 version. If the version number is final, that is considered a released version and not a prerelease. -->
-    <version>1.0.0</version>
+    <version>1.1.0</version>
     <packageSourceUrl>https://github.com/rtsoft-gmbh/up2date-win</packageSourceUrl>
-    <!-- owners is a poor name for maintainers of the package. It sticks around by this name for compatibility reasons. It basically means you. -->
     <owners>RTSoft</owners>
-    <!-- ============================== -->
-
-    <!-- == SOFTWARE SPECIFIC SECTION == -->
-    <!-- This section is about the software itself -->
     <title>RITMS UP2DATE</title>
     <authors>RTSoft</authors>
-    <!-- projectUrl is required for the community feed -->
     <projectUrl>https://ritms.online/up2date</projectUrl>
-    <!-- There are a number of CDN Services that can be used for hosting the Icon for a package. More information can be found here: https://docs.chocolatey.org/en-us/create/create-packages#package-icon-guidelines -->
-    <!-- Here is an example using Githack -->
     <iconUrl>https://raw.githubusercontent.com/rtsoft-gmbh/up2date-win/master/Up2dateService/Up2dateConsole/Images/Logo.ico</iconUrl>
     <copyright>2022 RTSoft</copyright>
-    <!-- If there is a license Url available, it is required for the community feed -->
     <licenseUrl>https://raw.githubusercontent.com/rtsoft-gmbh/up2date-win/master/LICENSE</licenseUrl>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
-    <!--<projectSourceUrl>Software Source Location - is the software FOSS somewhere? Link to it with this</projectSourceUrl>-->
-    <!--<docsUrl>At what url are the software docs located?</docsUrl>-->
-    <!--<mailingListUrl></mailingListUrl>-->
     <bugTrackerUrl>https://github.com/rtsoft-gmbh/up2date-win/issues</bugTrackerUrl>
     <tags>up2date</tags>
     <summary>Up2Date Client</summary>
     <description>RITMS UP2DATE is a cloud ready solution for unified software and firmware management. Use this for implementing lifecycle management for the full stack of drivers and firmware of connected devices.</description>
-
-    <!-- Specifying dependencies and version ranges? https://docs.nuget.org/create/versioning#specifying-version-ranges-in-.nuspec-files -->
-    <!--<dependencies>
+    <dependencies>
       <dependency id="dotnetfx" />
-    </dependencies>-->
+      <dependency id="vcredist140" />
```

Comment:
Still doesn't work for me:
"up2date not installed. An error occurred during installation:
 Unable to resolve dependency 'vcredist140'."